cmake/tg_owtConfig.cmake

```diff
@@ -2,5 +2,9 @@ if (@absl_FOUND@)
     include(CMakeFindDependencyMacro)
     find_dependency(absl REQUIRED)
 endif()
+if (@Crc32c_FOUND@)
+    include(CMakeFindDependencyMacro)
```

Comment:
I believe there's no real reason to include it conditionally and multiple-time include seems ugly, can you just move it to the start of the file with an empty line between it and conditions?

Reply:
done